lib/shared/bucketing/__test__/bucketing.test.ts

```diff
@@ -7,7 +7,7 @@ import {
     generateBucketedConfig,
     doesUserPassRollout
 } from '../src/bucketing'
-import { config, barrenConfig } from './data/testData'
+import { config, barrenConfig } from '@devcycle/bucketing-test-data/src/data/testData'
```

Comment:
is this not able to just import from the index anymore?

Reply:
no, because that would execute the writing to disk

Reply:
maybe you could do something like check for a command-line flag to enable the writing, and update the target definition to pass that flag?

Reply:
I don't think we can, because that would cause the files to export the TS code with the links to the Types library that the AS code was having issues with. We set up this whole build system so it only directly exports the JSON data.